miniapps/shifted_methods/distance.cpp

```diff
@@ -0,0 +1,285 @@
+//            MFEM Distance Function Solver - Parallel Version
```

Comment:
This is a very interesting mini app and I'd be interested to learn more about the method.  It would be great to have a short paragraph here describing the method and giving any relevant references.  I did see the reference to Kane et al in the header file but duplicating that here (or at least pointing people to the header) might be useful.